Tools/CISupport/ews-build/steps.py

```diff
@@ -4895,7 +4896,10 @@ def setLayoutTestCommand(self):
         # On the repeat steps we don't enable coredump generation (makes the run much slower if there are crashes)
         self.setCommand([arg for arg in self.command if arg != '--enable-core-dumps-nolimit'])
         first_results_failing_tests = set(self.getProperty('first_run_failures', []))
-        self.setCommand(self.command + ['--fully-parallel', '--repeat-each=%s' % self.NUM_REPEATS_PER_TEST] + sorted(first_results_failing_tests))
+        command = self.command + ['--fully-parallel', '--repeat-each=%s' % self.NUM_REPEATS_PER_TEST] + sorted(first_results_failing_tests)
+        if self.getProperty('platform') == 'gtk':
+            command = ['NUMBER_OF_PROCESSES=%s ' % self.NUMBER_OF_PROCESSES] + command
```

Comment:
There's a typo in the env var name: `NUMBER_OF_PROCESSORS`, even though we can also use `WEBKIT_TEST_CHILD_PROCESSES`.

Or, to align with the other options and avoid env vars at all, use `--child-processes=self.NUMBER_OF_PROCESSES`.
```suggestion
        command = self.command + ['--fully-parallel', '--repeat-each=%s' % self.NUM_REPEATS_PER_TEST]
        if self.getProperty('platform') == 'gtk':
            command.append(f'--child-processes={self.NUMBER_OF_PROCESSES}')
        command.extend(sorted(first_results_failing_tests))
```